brambling/models.py

```diff
@@ -1486,6 +1486,9 @@ def can_transfer(self):
             not self.transactions.filter(is_confirmed=False).exists()
         )
 
+    def get_transfer(self):
```

Comment:
I would prefer not to add another single-purpose method that has side effects (additional queries).